core/src/main/java/org/ironjacamar/core/CoreLogger.java

```diff
@@ -38,6 +39,40 @@
 @MessageLogger(projectCode = "IJ2")
 public interface CoreLogger extends BasicLogger
```

Comment:
Just checking... there are quite a few log messages added here that are not being used... I'm assuming there will be a future commit for using those?